contextutil/context.go

```diff
@@ -52,6 +43,19 @@ func TX(ctx context.Context) transactions.TxID {
 	return tid
 }
 
+// AccountDID extracts the AccountConfig DID from the given context value
+func AccountDID(ctx context.Context) (identity.DID, error) {
+	acc, err := Account(ctx)
+	if err != nil {
+		return identity.DID{}, err
```

Comment:
lets name the return values so that you dont have to recreate the DID for every return

Reply:
Actually in this scenario err would be shadowed from the named return. So I think it is cleaner to keep the explicit return.